runtime/gc_realtime/RealtimeMarkTask.cpp

```diff
@@ -68,13 +65,10 @@ MM_RealtimeMarkTask::cleanup(MM_EnvironmentBase *envBase)
 {
 	MM_EnvironmentRealtime *env = MM_EnvironmentRealtime::getEnvironment(envBase);
 	MM_GCExtensions *extensions = MM_GCExtensions::getExtensions(env);
-	GC_Environment *gcEnv = env->getGCEnvironment();
-	PORT_ACCESS_FROM_ENVIRONMENT(env);
-	
-	MM_GlobalGCStats *finalGCStats= &extensions->globalGCStats;
-	finalGCStats->markStats.merge(&env->_markStats);
-	extensions->markJavaStats.merge(&gcEnv->_markJavaStats);
-	finalGCStats->workPacketStats.merge(&env->_workPacketStats);
+	MM_MetronomeDelegate *delegate = envRealtime->getExtensions()->realtimeGC->getRealtimeDelegate();
```

Comment:
instead of envRealtime->getExtensions() you can use extensions declared line above